quantecon/game_theory/tests/test_mclennan_tourky.py

```diff
@@ -5,8 +5,12 @@
 
 """
 import numpy as np
-from nose.tools import ok_
+from numpy.testing import assert_allclose
+from nose.tools import ok_, raises
 from quantecon.game_theory import Player, NormalFormGame, mclennan_tourky
+from quantecon.game_theory.mclennan_tourky import _best_response_selection, \
+                                                  _flatten_action_profile, \
+                                                  _is_epsilon_nash
```

Comment:
Maybe this should be something like

```py
from quantecon.game_theory.mclennan_tourky import (
    _best_response_selection, _flatten_action_profile, _is_epsilon_nash
)
```

Reply:
thanks @oyamad I didn't know you could use ``(`` like this.